resources/js/components/entries/PublishForm.vue

```diff
@@ -12,6 +12,13 @@
             </h1>
 
             <dropdown-list class="mr-4" v-if="canEditBlueprint">
```

Comment:
One thing I've noticed is that the `v-if="canEditBlueprint"` here should probably be moved to the relevant `dropdown-item`?

Reply:
Ah yeah good spot.

Reply:
I moved that.